### PR TITLE
[TIMOB-19716] Support contact icon in UIApplicationShortcuts

### PR DESCRIPTION
**Pull request by hansemannn**
JIRA: https://jira.appcelerator.org/browse/TIMOB-19716

### CODE REVIEW

File: apidoc/Titanium/UI/iOS/ApplicationShortcuts.yml [hunk 1]
```diff
@@ -185,8 +247,9 @@ properties:
 
   - name: icon
     summary: |
-        The icon to be displayed on the application shortcut, for example <Titanium.UI.iOS.SHORTCUT_ICON_TYPE_COMPOSE>.
-        Note: You can also use a local image specified by the image path.
-    type: Number
+        The icon to be displayed on the application shortcut. You can either use one of the contastants like
```
Comment (cheekiatng) [inline apidoc/Titanium/UI/iOS/ApplicationShortcuts.yml]: spelling: not contestants, but constants

Comment (hansemannn) [inline apidoc/Titanium/UI/iOS/ApplicationShortcuts.yml]: Addressed your comment in the latest commit, thanks!

**Comment by cheekiatng** [comment]
CR and FT passed. 2 questions:
in your example app:
1. i tried doing "Add Shortcut" twice with 2 different contacts. but there's only 1 shortcut appearing. which is the first add. is this expected behavior?
2. Assuming it is the expected behavior, shouldn't the 2nd "add shortcut" overwrite the first? because the first contact always appears (until i chose remove)

**Comment by hansemannn** [comment]
The first button is "Add Ti.Contacts Application Shortcut" to create a shortcut and the second is "Remove Ti.Contacts Application Shortcut" to remove the previous created shortcut to allow the tester to test some contacts.

**Comment by cheekiatng** [comment]
APPROVED!

**Pull request merged**